Nos/Views/Settings/SettingsView.swift

```diff
@@ -217,6 +219,32 @@ struct SettingsView: View {
                     .padding(.vertical, 15)
             }
             .listRowGradientBackground()
+            
+            #if STAGING || DEBUG
+            if isDeleteAccountEnabled.wrappedValue {
```

Comment:
Unfortunately, with the existing pattern we have for the feature flag bindings, I wasn't able to make this view instantly update when you switch the toggle for enabling account deletion. You have to leave the view and come back. I tried about 10 different ways to remedy this, but I wasn't successful and had to time box it. I'm definitely interested if anybody has a better way to do this.

Reply:
I was able to get this to work with the new `@Observable` macro from iOS 17. I pushed a commit to your branch, check it out!

Reply:
Amazing! Thanks.